viewer/reveal.api.md

```diff
@@ -1422,6 +1422,20 @@ export class StyledPointCloudObjectCollection {
 // @public (undocumented)
 export type SupportedModelTypes = 'pointcloud' | 'cad';
 
+// @public
+export class ThickLine {
```

Comment:
Do we have to expose it externally? Since it is imported from utilities it should be fine being only internal, isn't it?

Reply:
The reason is that it's used by `MeasurementLineTool`. But I think I'll just create a new "ThickLine"-class and leave the measurement-line class as it is.